add possibility to define owner/group for local validation path and local challenge files (#48)
T-Systems-MMS/ansible-collection-acme: docs/http-challenge/local.md

# Variables for local http-challenge

| Variable                              | Required | Default       | Description
|---------------------------------------|----------|---------------|------------
| acme_local_validation_path            | no       | /var/www/html | Path where the validation-/ hashfiles get created

> | amce_local_validation_path_file_owner | no       | root          | User which own the validation-/ hash- files and path

Reviewer: ```suggestion
| amce_local_validation_path_file_owner | no       | root          | User who owns the validation-/ hash- files and path
```

> | amce_local_validation_path_file_owner | no       |               | User which own the validation-/ hash- files and path

Reviewer: ```suggestion
| acme_local_validation_path_file_owner | no       |               | User who owns the validation-/ hash- files and path
```

> | amce_local_validation_path_file_group | no       | root          | User Group which own the validation-/ hash- files and path 

Reviewer: ```suggestion
| amce_local_validation_path_file_group | no       | docker          | Group who owns the validation-/ hash- files and path 
```

> | amce_local_validation_path_file_group | no       |               | User Group which own the validation-/ hash- files and path 

Reviewer: ```suggestion
| acme_local_validation_path_file_group | no       |               | Group who owns the validation-/ hash- files and path 
```
